docs/pages/roadmap.js

```diff
@@ -0,0 +1,676 @@
+  {
+    'task': 'Generalized prop value change autofix',
+    'deadline': '',
+    'status': 'ok',
+    'platform': 'Web',
+    'description':
+      'Develop a lint rule to automatically change prop values of Gestalt component instances.',
+    'link': '',
+  },
+  {
+    'task': 'Testing helpers library',
+    'deadline': '',
+    'status': 'unstarted',
+    'platform': 'Web',
+    'description':
+      'Develop infrastructure to support Gestalt component integration testing in Cypress.',
+    'link': '',
+  },
+  {
+    'task': 'Visual diff testing',
+    'deadline': '',
+    'status': 'inProgress',
+    'platform': 'Web',
+    'description':
+      'Ship visual diff tests for all Gestalt components to guard against UI regressions.',
+    'link': '',
+  },
+];
+
+const inProgress = tasks.filter((x) => x.status === 'inProgress');
+
+const future = tasks.filter((x) => x.status === 'unstarted');
+
+const complete = tasks.filter((x) => x.status === 'ok');
+
+const LockIcon = function LockIcon({ size }: {| size: 12 | 14 |}) {
+  return (
+    <Tooltip text="Access is restricted to Pinterest employees" accessibilityLabel="" inline>
+      <TapArea rounding="circle" fullWidth={false}>
+        <Icon
+          accessibilityLabel="Access is restricted to Pinterest employees"
+          icon="lock"
+          size={size}
+        />{' '}
+      </TapArea>
+    </Tooltip>
+  );
+};
+
+function Task({
+  text,
+  description,
+  delivery,
+  platform,
+}: {|
+  text: string,
+  description: string,
+  delivery: string,
+  platform: string,
+|}) {
+  const title = platform ? `${text} (${platform})` : text;
+
+  const desc = (
+    <Box marginTop={-6}>
+      <Flex direction="column" gap={2}>
+        <Text>{description}</Text>
+        {delivery ? <Text size="100">Scheduled for {delivery}, 2022</Text> : null}
+      </Flex>
+    </Box>
+  );
+
+  return <MainSection.Subsection title={title}>{desc}</MainSection.Subsection>;
+}
+
+export default function RoadmapPage(): Node {
+  const theme1Items = inProgress.map((obj) => (
+    <Task
+      key={obj.task}
+      text={obj.task}
+      description={obj.description}
+      delivery={obj.deadline}
+      platform={obj.platform}
+    />
+  ));
+
+  const theme2Items = future.map((obj) => (
+    <Task
+      key={obj.task}
+      text={obj.task}
+      description={obj.description}
+      delivery={obj.deadline}
+      platform={obj.platform}
+    />
+  ));
+
+  const theme3Items = complete.map((obj) => (
+    <Task
+      key={obj.task}
+      text={obj.task}
+      description={obj.description}
+      delivery={obj.deadline}
+      platform={obj.platform}
+    />
+  ));
+
+  return (
+    <Page title="Upcoming work in 2022">
+      <PageHeader name="Upcoming work in 2022" showSourceLink={false} />
+
+      <Text>
+        The following reflects all public-facing work the Gestalt team plans to ship in 2022. To see
+        the full list of work our team plans to take on, please visit our&nbsp;
+        <Link
```

Comment:
for now, it might be worth bolding this to showcase it is a Link

Reply:
+1, the link icon looks very awkward currently.

Also this sentence needs a period at the end.